README.md

```diff
@@ -108,6 +108,17 @@ For example, the following example is invalid and will cause an error, because S
 As described above, `save.toml` is needed to configure tests. The idea is to have only one configuration file for a directory with tests (one to many relation). 
 Such directories we will call `test suites`. We decided to have only one configuration file as we have many times seen that for such tests there is a duplication of configuration in the same test suite.
 
+## Executing specific tests
```

Comment:
Let’s merge it with how to start section (rename it to something like: Usage)

Reply:
I didn't want to add it to `How to start`, because it's rather specific execution mode and I thought we should at first describe basic usage, files hierarchy and only then tell about individual test executions